app/charts/map/map-attribution.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import { Box, Grid } from "theme-ui";
+import { Icon } from "../../icons";
+
+const ATTRIBUTION = "© Data: MapTiler, OpenStreetMap contributors, Swisstopo";
+const ICON_SIZE = 14;
+
+interface Props {
+  compact: boolean;
+}
+
+export const MapAttribution = (props: Props) => {
+  const { compact } = props;
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  return !compact ? (
+    <MapAttributionBox>{ATTRIBUTION}</MapAttributionBox>
+  ) : (
+    <MapAttributionBox
+      onMouseEnter={() => setIsExpanded(true)}
+      onMouseLeave={() => setIsExpanded(false)}
+    >
+      <Grid
+        sx={{
+          gridTemplateColumns: `1fr ${ICON_SIZE}px`,
+          gridGap: 1,
+          alignItems: "flex-end",
+        }}
+      >
```

Comment:
Interesting, I would have used Stack for this.

```
<Stack direction='row' spacing={1}
</Stack>
```

I think it is a bit simpler and you do not need the ICON_SIZE.